app/models/supervisor_volunteer.rb

```diff
@@ -26,4 +32,4 @@ class SupervisorVolunteer < ApplicationRecord
 #
 #  fk_rails_...  (supervisor_id => users.id)
 #  fk_rails_...  (volunteer_id => users.id)
-#
+#
```

Comment:
I recommend telling VSCode to not trim line endings- something like this https://stackoverflow.com/questions/39525417/visual-studio-code-how-to-show-line-endings